addons/mail/models/discuss/ir_config_parameter.py

```diff
@@ -0,0 +1,19 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import re
```

Comment:
Don't put that code in discuss while original model is in mail. Keep things simple.